build/templates/values.yaml

```diff
@@ -435,6 +435,8 @@ tls:
     # The name of this ServiceAccount to use.
     # If not set and `create` is `true`, then a name is auto-generated.
     name: ""
+    # Additional serviceAccount annotations (e.g. for attaching AWS IAM roles to pods)
```

Comment:
the template got out of sync with the generated file.

(a good reason to get this dusted off 😅 https://github.com/cockroachdb/helm-charts/pull/282)